Sources/UnleashProxyClientSwift/Client/Bootstrap.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+public enum Bootstrap {
+    case toggles([Toggle])
+    case jsonFile(path: String)
+    
+    var toggles: [Toggle] {
+        switch self {
+        case .toggles(let toggles):
+            return toggles
+            
+        case .jsonFile(let path):
```

Comment:
I'm wondering if we could make it more similar to the android SDK where we pass a pre-existing file instead of a path to a file. Would it eliminate the fata error where file cannot be found and move it to the user space? From a quick check android SDK checks if the provided file exists and uses default when it doesn't.

Reply:
Ah I did want to do this but Swift doesn't really have a concept of a File object. However for the fatal error we could instead default to an empty list of toggles (a default). I wasn't sure whether to do that or not as since it may be misleading to a user that they input a file path and seemingly no issues reading from it even if invalid. Hence alerting the user earlier seemed more appropriate telling them the file they expected to exist is not actually on their filesystem. What do you think?

Reply:
maybe printing warning to the console that the file was not found and using a default empty list?

Reply:
Sure! I'll change that over and put some doc comment on it which should hopefully surface it :)